extensions/binding/org.eclipse.smarthome.binding.astro.test/pom.xml

```diff
@@ -4,36 +4,118 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.eclipse.smarthome.binding</groupId>
     <artifactId>pom</artifactId>
+    <groupId>org.eclipse.smarthome.binding</groupId>
     <version>0.10.0-SNAPSHOT</version>
   </parent>
 
-  <groupId>org.eclipse.smarthome.archetype</groupId>
+  <groupId>org.eclipse.smarthome.binding</groupId>
   <artifactId>org.eclipse.smarthome.binding.astro.test</artifactId>
-  <version>0.10.0-SNAPSHOT</version>
   <packaging>eclipse-test-plugin</packaging>
 
-  <name>Eclipse SmartHome Astro Binding Tests</name>
-
-  <properties>
-    <bundle.symbolicName>org.eclipse.smarthome.binding.astro.test</bundle.symbolicName>
-    <bundle.namespace>org.eclipse.smarthome.binding.astro</bundle.namespace>
-  </properties>
+  <name>Astro Binding Tests</name>
```

Comment:
why leave `Eclipse SmartHome` out here? I guess all other projects have this in their name.